### PR TITLE
Remove the slowdown on dufflebags but remove the extra size

### LINKED ISSUE #62: Dufflebag Encumbrance
Dufflebags have been and should continue to be a cosmetic choice. In prior versions of Hippie Station, they have been used just like any satchel or backpack. This needs to return, as Encumbrance of dufflebags is shitty as fuck, to put it unprofessionally. 

Some argue, "WELL THEY HAVE EXTRA SPACE SO THEY SHOULD HAVE A TRADE OFF"

My counter argument is, "Well then remove the extra space." Or don't. If you want to keep it and have that esoteric bit of knowledge that dufflebags are the definitive bag of choice, go ahead. 

Either way, FREE DUFFLEBAGS 2017

### PR DESCRIPTION
**Pull request by j-mie**
:cl: JamieH
tweak: Remove the slowdown on dufflebags but remove the extra size
/:cl:

Fixes #62

### CODE REVIEW

**Comment by ghost** [comment]
Good change, giving cosmetic items unneeded functionality is dumb. +1

**Comment by cacogen** [comment]
The entire point of the dufflebag is to hold extra items, with a trade-off.

As it is though, I never see the dufflebag get used unless it's one of the ones that come with items already in it.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -316,34 +316,33 @@
 
 /obj/item/weapon/storage/backpack/dufflebag
 	name = "dufflebag"
-	desc = "A large dufflebag for holding extra things."
+	desc = "A large dufflebag for holding things."
 	icon_state = "duffle"
 	item_state = "duffle"
-	slowdown = 1
-	max_combined_w_class = 30
+	max_combined_w_class = 21
```
Comment (ghost) [inline code/game/objects/items/weapons/storage/backpack.dm:322]: Isn't this unrequired since it would inherit the satchel size?

**Comment by KittyKathy** [comment]
We should probably just make dufflebags use the pre rebase sprite which was smaller and make a new dufflebag called something like a large dufflebag that uses this new sprite with slowdown and holding extra items because the sprite is pretty big compared to the other ones. Also you should only be able to pick the small dufflebag on the character preferences and have to find a large dufflebag in game if you want one.

*edited for typos

**Comment by Iamgoofball** [comment]
:-1:

the entire point of the dufflebags was for this

this is like removing the health bonus on armor and saying it's dumb for "cosmetic armor to have features tacked on"

**Comment by cacogen** [comment]
I think Kathy has a pretty good idea

**Comment by Iamgoofball** [comment]
kathy's idea is good do that one instead

**Pull request closed without merging**